README.md

```diff
@@ -88,102 +93,45 @@ Additionally, for the development we use:
 
 > See the [Development](#development) section for details on how to install `mockery`.
 
-# Quick-Start Installation
+# Quick-Start installation
 
-We provide an installation script to automatically install and update the latest version of Trento.
+We provide installations scripts for to automatically install and update the latest version of Trento.
+Please follow the installation in the given order/
 
-## Trento Server Installation
+## Trento Server installation
 
-The [packaging/helm](packaging/helm) directory contains the Helm chart for installing Trento Server in a Kubernetes cluster.
-
-### Install K3s
-
-If installing as root:
-
-```
-# curl -sfL https://get.k3s.io | sh
-```
-
-If installing as non-root user:
-
-```
-$ curl -sfL https://get.k3s.io | sh -s - --write-kubeconfig-mode 644
-```
-
-Export KUBECONFIG env variable:
-
-```
-$ export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
-```
-
-Please refer to the [K3s official documentation](https://rancher.com/docs/k3s/latest/en/installation/) for more information about the installation.
-
-### Install Helm and chart dependencies
-
-Install Helm:
-
-```
-$ curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
-```
-
-Please refer to the [Helm official documentation](https://helm.sh/docs/intro/install/) for more information about the installation.
-
-### Install the Trento Server Helm chart
-
-Add HashiCorp Helm repository:
-
-```
-$ helm repo add hashicorp https://helm.releases.hashicorp.com
-$ helm repo update
-```
-
-Install chart dependencies:
-
-```
-$ cd packaging/helm/trento-server/
-$ helm dependency update
-```
-
-The runner component of Trento server needs ssh access to the agent nodes to perform the checks.
-
-You need to pass a valid private key used for SSH authentication to the Helm chart, and it will be stored
-in the K3s cluster as a secret.
+The script installs a single node K3s cluster and uses the [trento-server Helm chart](packaging/helm/trento-server)
+to bootstrap a complete Trento server component.
 
-Please refer to the [Trento Runner](#trento-runner) for more information.
-
-Install Trento server chart:
+You can `curl | bash` if you want to live on the edge.
 
 ```
-$ helm install trento . --set-file trento-runner.privateKey=/your/path/id_rsa_runner
+$ curl -sfL https://raw.githubusercontent.com/trento-project/trento/main/install-server.sh | bash
```

Comment:
Can this work without providing the ssh key path?

Reply:
The user will be prompted when not providing the arguments through the command line. IMHO that's fine, right?

Reply:
Yes, it works. I most probably did something wrong the last time as it was not promptying anything... XD